science/digital_rf/Portfile

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           cxx11 1.1
+PortGroup           python 1.0
+
+github.setup        MITHaystack digital_rf 2.6.0
+name                digital_rf
+version             2.6.0
+maintainers         @MITHaystack {mit.edu:rvolz @ryanvolz} {mit.edu:swoboj @jswoboda}
+description         Read, write, and interact with data in the Digital RF and Digital Metadata formats.
+long_description    Tools to store RF and metadata for scientific engineering research projects. 
+categories          science
+license             BSD
+platforms           darwin
+revision            0
+dist_subdir         digital_rf
+homepage            https://github.com/MITHaystack/digital_rf
+
+checksums           rmd160  f095ff1a80f7175f57a45048c826fe68b6dd56e9 \
+                    sha256  6468418a06c269964c9a5c5f145f7bfebf951226e3a85977226c7967fa50e691
+
+configure.ldflags-delete -L${prefix}/lib
+
+# dependencies
+depends_lib-append port:uhd
+depends_lib-append port:boost
+depends_lib-append port:gnutls
+depends_lib-append port:gnuradio
+depends_lib-append port:py27-mako
+depends_lib-append port:python27
+depends_lib-append port:py27-numpy
+depends_lib-append port:py27-matplotlib
+depends_lib-append port:py27-scipy
+depends_lib-append port:py27-pandas
+depends_lib-append port:py27-pkgconfig
+depends_lib-append port:hdf5
+depends_lib-append port:py27-packaging
+depends_lib-append port:py27-dateutil
+depends_lib-append port:py27-h5py
+depends_lib-append port:py27-six
+depends_lib-append port:py27-tz
+depends_lib-append port:py27-watchdog
+
```

Comment:
You can do all of these with one depends_lib-append by doing something like...

```
depends_lib-append port:uhd \
                                    port:boost \
...
```

Also, you should sort the dependencies, it will make it easier to find things.

Given that you're depending on a bunch of py27 ports, you need not depend on python27 (which isn't a library anyway).

I would suggest that, if you can, depend on python 3.6 ports instead, as python 2.7 is approaching end of life.